lls/main.c

```diff
@@ -547,6 +572,14 @@ assign_lls_queue_ids(struct lls_config *lls_conf)
 			goto fail;
 		lls_conf->rx_queue_back = ret;
 
+		if (lls_conf->net->back.rss) {
+			ret = get_queue_id(&lls_conf->net->back, QUEUE_TYPE_RX,
```

Comment:
You forgot to remove the code that this new `if` statement is protecting above, that is, the code above and inside the body of the new `if` statement are duplicated.